go_expvar/README.md

```diff
@@ -77,3 +79,4 @@ Need help? Contact [Datadog Support][9].
 [8]: https://github.com/DataDog/integrations-core/blob/master/go_expvar/metadata.csv
 [9]: http://docs.datadoghq.com/help/
 [10]: https://www.datadoghq.com/blog/instrument-go-apps-expvar-datadog/
+[11]: http://raw.githubusercontent.com/DataDog/documentation/master/src/images/integrations/goexpvar/go_graph.png
```

Comment:
Ah, so the images are failing to load on Github markdown preview because this should be `https not `http`.